tests/transformers/test_transformers_model_export.py

```diff
@@ -1713,6 +1713,176 @@ def test_qa_pipeline_pyfunc_predict(small_qa_pipeline, tmp_path):
 
     assert values.to_dict(orient="records") == [{0: "Run"}]
 
+    inference_payload = json.dumps(
+        {
+            "inputs": {
+                "question": ["What color is it?", "How do the people go?"],
+                "context": [
+                    "Some people said it was green but I know that it's pink.",
+                    "The people on the bus go up and down. Up and down.",
+                ],
+            }
+        }
+    )
+    response = pyfunc_serve_and_score_model(
+        model_uri,
+        data=inference_payload,
+        content_type=pyfunc_scoring_server.CONTENT_TYPE_JSON,
+        extra_args=["--env-manager", "local"],
+    )
+    values = PredictionsResponse.from_json(response.content.decode("utf-8")).get_predictions()
+
+    assert values.to_dict(orient="records") == [{0: "pink"}, {0: "up and down"}]
+
+
+def test_classifier_pipeline_pyfunc_predict(text_classification_pipeline, tmp_path):
+    artifact_path = "text_classifier_model"
+    with mlflow.start_run():
+        mlflow.transformers.log_model(
+            transformers_model=text_classification_pipeline,
+            artifact_path=artifact_path,
+        )
+        model_uri = mlflow.get_artifact_uri(artifact_path)
+
+    inference_payload = json.dumps(
+        {
+            "inputs": [
+                "I think this sushi might have gone off",
+                "That gym smells like feet, hot garbage, and sadness",
+                "I love that we have a moon",
+            ]
+        }
+    )
+
+    from mlflow.deployments import PredictionsResponse
```

Comment:
Looks like we import this in each test. Can we import this at the top?

Reply:
moved!